dev/com.ibm.ws.sipcontainer/bnd.bnd

```diff
@@ -25,6 +25,7 @@ Export-Package: \
   com.ibm.websphere.sip.*, \
   com.ibm.ws.sip.container.pmi.*, \
   com.ibm.ws.sip.container.was, \
+  com.ibm.sip.util.log;version=1.0.89, \
```

Comment:
Exported to use logging in the 3.1   bundle.

Reply:
Where did the version come from?

Reply:
Bnd gave me these version to export.  It's based on the current bundle version (i.e. `wlp/lib/com.ibm.ws.sipcontainer_1.0.89.jar`)